### ISSUE TITLE
[v5] On Windows Docker report portal is getting bad gateway error

### ISSUE THREAD

**Issue by vishallanke** [reportportal/reportportal#1133]
I am installing report portal on Windows using Docker. [Link](https://reportportal.io/docs/Deploy-ReportPortal-with)
Installation gets successful without any errors.
I tried to open report portal - `http://127.0.0.1:8080`. UI gets opened.
After entering default credential, I am getting error - "502 Bad Gateway, An error occurred while connecting to server "
Note - My machine is in Corporate network with Automated Configuration Script is configured inside Internet Options.

```
**Enter Credentials and Click on Login
Check Network Call
Request URL: http://127.0.0.1:8080/uat/sso/oauth/token
Request Method: POST
Status Code: 502 Bad Gateway
Remote Address: 127.0.0.1:8080**

```

**Comment by evjlobanova** [comment]
We  strongly recommend installing RP on Linux based OS 
@Cyberglamdring Hleb, could you please check what can be wrong?

**Comment by Aravamudhan** [comment]
I am also facing the same issue. My OS is windows 10 enterprise edition and my docker version is "Docker version 19.03.12, build 48a66213fe". I have installed "Docker Desktop" and using docker through that. I have allocated 4GB to docker and hello-world image from dockerhub is working correctly. The images get pulled correctly. Even the login works as it says logged in successfully, but the /api/v1/user endpoint fails. Because of that the app does not proceed.

The network calls

<img width="960" alt="report_portal_login_issue" src="https://user-images.githubusercontent.com/11723195/93571287-da95cf00-f9b1-11ea-999c-64a3fc1104e3.png">

The status of the containers
![rp_issue](https://user-images.githubusercontent.com/11723195/93571595-50019f80-f9b2-11ea-99c5-b3724731e209.jpg)

Like @vishallanke says above the endpoint http://127.0.0.1:8080/uat/sso/oauth/token starts failing too.
